stable/influxdb/templates/post-install-create-db.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.createDatabase.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  name: {{ template "fullname" . }}-create-db
+  annotations:
+    "helm.sh/hook": post-install
+spec:
+  activeDeadlineSeconds: {{ default 300 .Values.createDatabase.activeDeadlineSeconds }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}-create-db
+        image: "{{ .Values.createDatabase.image }}"
+        env:
+          - name: INFLUXDB_DATABASE
+            value: {{ default "influxdb" .Values.createDatabase.database.database | quote }}
+          - name: INFLUXDB_DURATION
+            value: {{ default "3d" .Values.createDatabase.database.duration | quote }}
```

Comment:
The default retention duration in InfluxDB is `0d` which is for infinite retention. Please change this default to the standard default on the database.